pkg/infra/appcontext/user.go

```diff
@@ -50,13 +51,21 @@ func User(ctx context.Context) (*user.SignedInUser, error) {
 			case k8suser.APIServerUser:
 				fallthrough
 			case k8suser.SystemPrivilegedGroup:
+				orgId := int64(1)
```

Comment:
i'm curious how will we get this to work with multiple orgs?

Reply:
regular users have their orgId set on connection.  This hook is just for the special k8s privileged user that has access to everything.

For "normal" access every user has an org

Is the question, how can a user query across multiple orgs?  That is not supported by our existing APIs, so 🤷🏻 multi-tentnat/multi-org controllers will either need to wait for unified storage that will support that as a first class operation, or we will need to update our commands to support it.